src/lib/bootloader/finish_client.rb

```diff
@@ -44,6 +44,7 @@ def write
 
       # we do not manage bootloader, so relax :)
       return true if bl_current.name == "none"
+      return true if Yast::BootStorage.boot_filesystem.is?(:nfs)
```

Comment:
Isn't this somehow redundant with the previous condition? I mean, if `boot_filesystem.is?(:nfs)` couldn't we be quite sure than `bl_current.name == "none"` (according to `ProposalClient#make_proposal`)

Reply:
It should be redundant, it was the first defensive check I added, before proposing a "none" bootloader.